src/reporting/Market.sol

```diff
@@ -444,24 +461,29 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
             return ReportingState.FORKING;
         }
 
+        if (block.timestamp > reportingWindow.getEndTime()) {
+            return ReportingState.AWAITING_FINALIZATION;
```

Comment:
I wonder if we should rename this state since now it means (if I am following the code right) `EITHER_AWAITING_FINALIZATION_OR_MIGRATION_TO_NEW_WINDOW` (note: I don't actually recommend that as an enum name).

Reply:
Alternatively, can we differentiate between the two easily and have two different states?

Reply:
I would really like to have a distinct phase for this case (No reporters reported on the market during reporting). It does mean that there will be two phases which indicate the market needs to have `tryFinalize` called, but that's probably fine as long as the name reflects that.

Maybe something like `AWAITING_FINALIZATION_MIGRATION`?